charts/vmclarity/templates/apiserver/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ printf "%s-apiserver" (include "vmclarity.names.fullname" .) }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "vmclarity.labels.standard" . | nindent 4 }}
```

Comment:
wouldn't it be easier to create an `_helper.tpl` for each component with the full name, the labels, etc..?
I see there are few places we are using the same stuff.

Reply:
+1 I really like this, I'll do that